src/backups.py

```diff
@@ -299,12 +299,18 @@ def _initialise_stanza(self) -> None:
             self.charm.unit.status = BlockedStatus(FAILED_TO_INITIALIZE_STANZA_ERROR_MESSAGE)
             return
 
+        self.start_stop_pgbackrest_service()
+
         # Store the stanza name to be used in configurations updates.
-        self.charm.app_peer_data.update({"stanza": self.stanza_name})
+        self.charm.app_peer_data.update({"stanza": self.stanza_name, "init-pgbackrest": "True"})
 
+    def check_stanza(self) -> None:
+        """Runs the pgbackrest stanza validation."""
         # Update the configuration to use pgBackRest as the archiving mechanism.
         self.charm.update_config()
 
+        self.charm.unit.status = MaintenanceStatus("checking stanza")
```

Comment:
Do we really need this status? I think it is ok just put "Checking stanza" into logs.

Reply:
We don't need it, but I think it makes things clearer to the user.